scripts/archive/make_hasura_configurations.sh

```diff
@@ -1,20 +0,0 @@
-#!/bin/bash
```

Comment:
I misspoke in the thread; I think we'll want to keep these scripts around, and just delete the OCaml code in this directory.